lib/sms_autofill.dart

```diff
@@ -391,7 +392,7 @@ mixin CodeAutoFill {
   }
 
   Future<void> cancel() {
-    return _subscription?.cancel();
+    return _subscription!.cancel();
```

Comment:
listenForCode is not necessarily called, that why `_subscription ` can be null, here that's the correct behavior

Reply:
I understand why you did it, but it will be better to put `async` in the signature to have always a Future returned.

Reply:
```
Future<void> cancel() async {
     await _subscription?.cancel();
}
```
That should work no ?